### ISSUE TITLE
No weapon viewmodel on Vulkan Renderer

### ISSUE THREAD

**Issue by Scattershor** [dpjudas/UT99VulkanDrv#35]
![image](https://github.com/dpjudas/UT99VulkanDrv/assets/58623117/1acaf0ce-299a-42de-a176-bcde54690f38)

CPU: 5800X3D
GPU: 7900XTX

**Comment by dpjudas** [comment]
I am not able to reproduce this on my computer - even tried a 4:3 resolution like your screenshots suggest and the weapon showed up just fine. Are you running any mods for UT? Is this the latest driver for the 7900XTX?

**Comment by Scattershor** [comment]
Yes, I am using the latest driver. I tried running many resolutions, but no change. The D3D12 & 11 renderers work with no issue.
________________________________
